tslint.json

```diff
@@ -15,6 +15,7 @@
         "member-ordering": [
             false
         ],
-        "interface-over-type-literal": false
+        "interface-over-type-literal": false,
+        "no-console": false
```

Comment:
Do not disable these linting rules globally.
You can disable a specific rule for a specific line.

This is preferrable to disabling rules globally, because you make it clear that a certain line does not comply with the standards, but there is a good reason for it.

We don't want UI components to start writing things to the console, and this linting rule helps us with that.

Reply:
Oh I see, didn't know you could do that